docs/integrations/persisting-store-data.md

```diff
@@ -571,3 +571,34 @@ export const useBearStore = create<MyState>()(
   )
 )
 ```
+
+### How do I use deal with Map/Set with the persist api
+
+With the previous persist API, you would use `serialize`/`deserialize` to deal with Map and Set and convert them into Array so they could be parsed into proper JSON.
```

Comment:
If I understood it correctly, this is how I'd rephrase it.

```suggestion
With the previous persist API, you would use `serialize`/`deserialize`
to deal with mapping and setting the data and converting it
into an Array, so it could be parsed into a proper JSON.
```

Reply:
Not really, i'm talking about the JS Object `Map` and `Set` not mapping and setting.

Reply:
Alright, you can wrap it in backticks, so `Map` and `Set`, so that it's clear that they're JS objects.

Reply:
And you can split this long line to shorter ones, to match the rest of this file.